supporting_scripts/hyperion/consistency-check-benchmark/report.py

```
@@ -97,9 +99,197 @@ def summarize_report(report_md_path: str, summary_md_path: str) -> None:
+    if match:
+        repo_slug = match.group(1)
+        raw_base = f"https://raw.githubusercontent.com/{repo_slug}/{branch_name}/results/{DATASET_VERSION}/{approach_id}"
+        pr_body = re.sub(
+            r"!\[([^\]]*)\]\((?!https?://)([^)]+)\)",
+            lambda m: f"![{m.group(1)}]({raw_base}/{m.group(2)})",
+            pr_body,
+        )
+
+    try:
+        # Check if the branch already exists
+        branch_check = subprocess.run(
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)